webpack.config.js

```diff
@@ -44,5 +46,18 @@ module.exports = async function (env, argv) {
       }),
     )
   }
+  if (process.env.SENTRY_AUTH_TOKEN) {
+    config.plugins.push(
+      sentryWebpackPlugin({
+        org: 'blueskyweb',
+        project: 'app',
+        authToken: process.env.SENTRY_AUTH_TOKEN,
+        release: {
+          name: process.env.SENTRY_RELEASE || version,
+          dist: process.env.SENTRY_DIST,
+        },
+      }),
+    )
```

Comment:
Just following the [Sentry docs](https://docs.sentry.io/platforms/javascript/guides/react/sourcemaps/uploading/webpack/) for integrating with Webpack. Confirmed this works with source maps 👍